ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -285,10 +282,11 @@ post_provision_config_nodes() {
     else
         cmd+=(upgrade)
     fi
-    if ! "${cmd[@]}" --exclude golang-*.daos.*; then
+    if ! "${cmd[@]}"; then
         dump_repos
         return 1
     fi
+    rpm -qa | grep ^go
```

Comment:
debugging left in?

Reply:
Not really.  It's an added ability to validate which version of go packages are installed on a node in case anyone is questioning it.